source/agora/node/Ledger.d

```diff
@@ -419,13 +419,18 @@ public class Ledger
 
     ***************************************************************************/
 
-    public Hash[] getMerklePath (ulong block_height, Hash hash) @safe
+    public Hash[] getMerklePath (ulong block_height, Hash hash) @safe nothrow
     {
         if (this.getBlockHeight() < block_height)
             return null;
 
         Block block;
-        this.storage.readBlock(block, block_height);
+        if (!this.storage.tryReadBlock(block, block_height))
+        {
+            log.trace("Could not read block at height {}: {}");
```

Comment:
`log.trace("Could not read block at height {}: {}",block_height,  block);`
Is it okay if It don't have it?

Reply:
Oops

Reply:
Good eyes @TrustHenry.